test/models/pregnancy_test.rb

```diff
@@ -43,4 +43,23 @@ class PregnancyTest < ActiveSupport::TestCase
       assert @pregnancy.created_by
     end
   end
+
+  before do
+    @pregnancy.dcaf_soft_pledge = 500
+    @pt_1.appointment_date = DateTime.now + 14
+    @clinic = create :clinic
+  end
+
+  describe 'pledge_sent' do
+    it 'should update the pledge_sent field to true' do
+      @pregnancy.pledge_sent = true
+      assert_equal true, @pregnancy.pledge_sent
+    end
+
+    it 'should not update pledge_sent field to true if the DCAF soft pledge amount is nil' do
+      @pregnancy.dcaf_soft_pledge = nil
+      @pregnancy.pledge_sent = true
+      assert_equal nil, @pregnancy.pledge_sent
```

Comment:
Similar deal here -- `pledge_sent` is going to always return true here because we haven't actually checked for validity yet. Swap the assertion here out to `refute @pregnancy.valid?`